Tests/PitchSpellerTests/CrossTests.swift

```diff
@@ -0,0 +1,32 @@
+//
+//  CrossTests.swift
+//  PitchSpellerTests
+//
+//  Created by Benjamin Wetherfield on 7/1/18.
+//
+
+import XCTest
+@testable import PitchSpeller
+
+class CrossTests: XCTestCase {
+    
+    func testEqualityFunction() {
+        let pair1 = Cross<Int, String>(3, "three")
```

Comment:
I could buy the argument that it could be helpful to another human to explicitly declare the generic type members here (`<Int,String>`) — especially in more intricate typing scenarios — but I am relatively certain that the compiler will infer them if you just put: `Cross(3, "three")`.

Might be worth seeing if that's true and trimming these all down. That being said, super micro nit that is not required at all.

Reply:
Yeah, interesting